### PR TITLE
Thread metrics calculation

### PR DESCRIPTION
**Pull request by michielderoos**
**Describe the Pull Request**

All metrics now calculate at one-- it's much faster now!  

**Todo**

- [x] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes, checking [accessibility](https://www.a11yproject.com/checklist/) etc.
- [x] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver
- [ ] Add release notes to [unreleased changelog](https://github.com/teamsempo/SempoBlockchain/blob/master/CHANGELOG.md#unreleased)

### CODE REVIEW

**Comment by michielderoos** [comment]
[Accelerate metrics](https://trello.com/c/ezLIDVes/850-accelerate-metrics)

**Comment by michielderoos** [comment]
[Accelerate metrics](https://trello.com/c/ezLIDVes/850-accelerate-metrics)

**Comment by michielderoos** [comment]
[Accelerate metrics](https://trello.com/c/ezLIDVes/850-accelerate-metrics)

File: .circleci/config.yml [hunk 1]
```diff
@@ -237,6 +237,7 @@ workflows:
             branches:
               only:
                 - master
+                - thread-metrics-calculation
```
Comment (tristanhcole) [inline .circleci/config.yml:240]: Remove before merge

**Review by tristanhcole — APPROVED** [review]
Tested and LGTM at a high level, although, tbh I don't understand session's enough to have a strong opinion on this.

Just checking this doesn't impact how we are handling global sessions? https://docs.sqlalchemy.org/en/14/orm/contextual.html#thread-local-scope

**Pull request merged**